components/webhook-mgt/org.wso2.carbon.identity.webhook.management/src/main/java/org/wso2/carbon/identity/webhook/management/api/model/WebhookDTO.java

```diff
@@ -0,0 +1,406 @@
+     */
+    public WebhookStatus getStatus() {
+
+        return status;
+    }
+
+    /**
+     * Get webhook status as string.
+     *
+     * @return Status string.
+     */
+    public String getStatusString() {
+
+        return status != null ? status.name() : null;
+    }
+
+    /**
+     * Builder class for WebhookDTO.
+     */
+    public static class Builder {
+
+        private String id;
+        private String createdAt;
+        private String updatedAt;
+        private String endpoint;
+        private String eventSchemaName;
+        private String eventSchemaUri;
+        private String description;
+        private List<String> eventsSubscribed = new ArrayList<>();
+        private WebhookStatus status;
+
+        /**
+         * Default constructor.
+         */
+        public Builder() {
+
+        }
+
+        /**
+         * Create a builder from an existing WebhookDTO.
+         *
+         * @param webhookDTO The webhook DTO to copy from.
+         * @return Builder instance.
+         */
+        public static Builder fromWebhookDTO(WebhookDTO webhookDTO) {
+
+            Builder builder = new Builder();
+
+            if (webhookDTO == null) {
+                return builder;
+            }
+
+            builder.id = webhookDTO.getId();
+            builder.createdAt = webhookDTO.getCreatedAt();
+            builder.updatedAt = webhookDTO.getUpdatedAt();
+            builder.endpoint = webhookDTO.getEndpoint();
+            builder.eventSchemaName = webhookDTO.getEventSchemaName();
+            builder.eventSchemaUri = webhookDTO.getEventSchemaUri();
+            builder.description = webhookDTO.getDescription();
+
+            if (webhookDTO.getEventsSubscribed() != null) {
+                builder.eventsSubscribed = new ArrayList<>(webhookDTO.getEventsSubscribed());
+            }
+
+            builder.status = webhookDTO.getStatus();
+            return builder;
+        }
+
+        /**
+         * Create a builder from an existing Webhook entity.
+         *
+         * @param webhook The webhook entity to convert.
+         * @return Builder instance configured with webhook data.
+         */
+        public static Builder fromWebhook(Webhook webhook) {
+
+            Builder builder = new Builder();
+
+            if (webhook == null) {
+                return builder;
+            }
+
+            builder.id = webhook.getId();
+
+            if (webhook.getCreatedAt() != null) {
+                Instant instant = webhook.getCreatedAt().toInstant();
+                ZonedDateTime zdt = instant.atZone(ZoneId.of("UTC"));
+                builder.createdAt = zdt.format(DateTimeFormatter.ISO_INSTANT);
+            }
+
+            if (webhook.getUpdatedAt() != null) {
+                Instant instant = webhook.getUpdatedAt().toInstant();
+                ZonedDateTime zdt = instant.atZone(ZoneId.of("UTC"));
+                builder.updatedAt = zdt.format(DateTimeFormatter.ISO_INSTANT);
```

Comment:
Is this consistent with other places where we return created and updated time. for example in scim api

Reply:
used java.sql.ResultSet.gettimestamp method